iceoryx_utils/platform/mac/include/iceoryx_utils/platform/mqueue.hpp

```diff
@@ -14,4 +14,58 @@
 
 #pragma once
 
-#include <mqueue.h>
+#include "iceoryx_utils/platform/types.hpp"
```

Comment:
We do not use this but the unix domain socket for Mac. So we do not need this file but have to redirect from message queue to unix domain socket, or?

Reply:
You are completely right but for the moment I require the stub file to continue my work, otherwise it would not even compile on mac. When we switched to sockets this file goes down the drain...